libs/notification/src/account-notification.service.ts

```diff
@@ -39,4 +39,36 @@ export class AccountNotificationService extends TypeOrmCrudService<AccountNotifi
       isArchived: updateDto.isArchived,
     });
   }
+
+  async readAccountNotifications(
+    accountId: string,
+  ): Promise<AccountNotification[]> {
+    const accountNotifications = await this.accountNotificationRepository.find({
+      accountId,
+      isRead: false,
+    });
+
+    return this.accountNotificationRepository.save(
```

Comment:
Can we user 'upset'-like calls here to avoid 1 more call invocation.

Reply:
Good point, done